boards/shared/Src/Io/Io_SharedAdc.c

```diff
@@ -7,6 +7,15 @@ float Io_SharedAdc_ConvertRawAdcValueToVoltage(
 {
     uint32_t full_scale;
 
+    float single_diff_scaling_ratio = 6.6f;
+    // TODO: Add switch statement that grabs adc single/diff setting and change
```

Comment:
we can probably do this for the current PR

Reply:
you will need to explain to me how to access variables from cube in our code